oct/config.sh.in

```diff
@@ -0,0 +1,13 @@
+AWK=@AWK@
+BZIP2=$(which bzip2)
+FILE=$(which file)
+GZIP=@GZIP@
+DEFLATE="./minideflate"
+MINIGZ="@srcdir@/minigzipsh"
```

Comment:
This means you need `minigzipsh` on your srcdir to be able to run `make check` and have all tests passing.
It's confusing you need external tools on the source directory to be able to run unit tests.
IMHO it would be better to keep `oct` out of `make check` or to generate the `minigzipsh` on the config.sh.

Reply:
@lucmaga I've just pushed a new version that removes `oct` from `make check`.